jenkins/ci.suse.de/cloud-mkphyscloud-qa-scenario-2a.yaml

```diff
@@ -116,7 +116,7 @@
 
       - string:
           name: commands
-          default: prepareinstallcrowbar bootstrapcrowbar installcrowbar allocate setup_aliases waitcloud
+          default: prepareinstallcrowbar bootstrapcrowbar installcrowbar allocate  waitcloud setup_aliases
```

Comment:
you have 2 spaces before waitcloud. looking a bit ugly. same in the other file.
However, commit messages should tell about why this change is needed.